style.css

```diff
@@ -6,468 +6,975 @@
+    max-width: 100%;
+    flex-direction: column;
+  }
+
+  .prof p {
+    color: #b3bac5;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 19px;
+    margin: 20px;
+    line-height: 28px;
+    text-align: center;
+    font-family: 'Roboto', Arial, Helvetica, sans-serif;
+  }
+
+  .lang-skills {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-between;
+    align-items: center;
+    padding: 20px;
+    gap: 30px;
+  }
 
-.mail-line {
-  display: block;
-  border-top: 3px solid rgb(0, 0, 0);
-  margin: 0 0 7px 0;
-  width: 174px;
+  .lang-skills .lang {
+    flex-grow: 0;
+    display: flex;
+    flex-direction: column;
+  }
+
+  h5 {
+    font-size: 18px;
+    font-weight: 200;
+    color: cornflowerblue;
+    padding-left: 2px;
+    font-family: 'Roboto', Arial, Helvetica, sans-serif;
+  }
+
+  .lang ul,
+  li {
+    color: #b3bac5;
+    font-size: 15px;
+    padding: 0 2px;
+    font-weight: 400;
+    line-height: 27px;
+    list-style-type: none;
+    font-family: 'Roboto', Arial, Helvetica, sans-serif;
+  }
+
+  .end-line {
+    display: block;
+    align-self: center;
+    border-bottom: 2px solid rgb(70, 67, 67);
+    margin: 10px 0 0 0;
+    width: 100%;
+  }
+
+  .back-foot {
+    background-image: url('./images/circle\ 2\ hder@2x.png');
+    background-position: relative;
+    background-size: 17%;
+    background-repeat: no-repeat;
+    background-position-x: right;
+    background-position-y: bottom;
+  }
+
+  .footer {
+    display: flex;
+    height: 158px;
+    width: 100%;
+    position: relative;
+    justify-content: center;
+    align-content: center;
+    padding: 50px 0;
+    margin-bottom: 15px;
+  }
+
+  .media-icons {
+    display: flex;
+    align-items: center;
+    position: relative;
+  }
+
+  .footer .fa {
+    text-align: center;
+    margin: 17px 4px;
+    font-size: 30px;
+    width: 26px;
+    height: 28;
+    color: #b3bac5;
+    background: #00005b;
+    text-decoration: none;
+  }
+
+  .footer li {
+    display: flex;
+    list-style-type: none;
+  }
+
+  /* Portfolio-Work section */
+
+  .work-content {
+    display: flex;
+    flex-direction: column;
+    padding: 22px;
+    margin-block: 25px;
+  }
+
+  .recent-work > h2 {
+    display: inline-block;
+    color: #6495ed;
+    margin-inline: 7px;
+    margin-block: 5px;
+    font-size: 40px;
+    font-weight: bold;
+    font-family: 'Roboto', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+  }
+
+  .recent-work > .wh2 {
+    margin-bottom: 25px;
+  }
+
+  .work-1 {
+    display: flex;
+    flex-direction: row;
+    gap: 2rem;
+    margin-block: 40px;
+  }
+
+  .work-2 {
+    display: flex;
+    flex-direction: row-reverse;
+    gap: 2rem;
+    margin-block: 40px;
+  }
+
+  .multi-post {
+    max-width: 37%;
+    height: auto;
+  }
+
+  .image {
+    width: 63%;
+    height: auto;
+  }
+
+  .image > img {
+    width: 100%;
+    max-height: auto;
+  }
+
+  .multi-post > h3 {
```

Comment:
- [ ] Kindly increase the font weight of the "Multi-post stories" headline . Consider increasing it to  `700` as specified in the figma template.
 
Figma Design             |  Your Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/60618877/178802708-cdc2f21c-b137-436c-96bb-38dab537b67a.png)  |  ![image](https://user-images.githubusercontent.com/60618877/178802562-c0f8d320-2941-48c2-99ec-f09e10d5b442.png)